sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClient.java

```diff
@@ -123,60 +123,6 @@ public DetectedLanguage detectLanguage(String document, String countryHint) {
         return client.detectLanguage(document, countryHint).block();
     }
 
-    /**
-     * Detects Language for a batch of documents.
-     *
-     * This method will use the default country hint that sets up in
-     * {@link TextAnalyticsClientBuilder#defaultCountryHint(String)}. If none is specified, service will use 'US' as
-     * the country hint.
-     *
-     * <p><strong>Code Sample</strong></p>
-     * <p>Detects the languages in a list of documents.</p>
-     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsClient.detectLanguageBatch#Iterable}
-     *
-     * @param documents The list of documents to detect languages for.
-     * For text length limits, maximum batch size, and supported text encoding, see
-     * <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits">data limits</a>.
-     *
-     * @return A {@link TextAnalyticsPagedIterable} contains a list of
-     * {@link DetectLanguageResult detected language document result}.
-     *
-     * @throws NullPointerException if {@code documents} is {@code null}.
-     * @throws IllegalArgumentException if {@code documents} is empty.
-     */
-    @ServiceMethod(returns = ReturnType.COLLECTION)
-    public TextAnalyticsPagedIterable<DetectLanguageResult> detectLanguageBatch(Iterable<String> documents) {
```

Comment:
Why are these overloads removed?

Reply:
as discussed offline, we reduce the overload APIs to one